doc/jsoo.rst

```diff
@@ -1,8 +1,8 @@
 .. _jsoo:
 
-**********************
-JavaScript Compilation
-**********************
+***************************************
+JavaScript Compilation With Js_of_ocaml
+***************************************
 
 Js_of_ocaml_ is a compiler from OCaml to JavaScript. The compiler works by
```

Comment:
```suggestion
Js_of_ocaml is a compiler from OCaml to JavaScript. The compiler works by
```
removed extra space

Reply:
But, wouldn't this remove the link from the rendered document? I am reusing the link anchor created here:

https://github.com/ocaml/dune/blob/dd9fc68d4c09ea0236777f61ea96fb3e17e58b1a/doc/jsoo.rst#L72

Maybe it'd be better if I use inline links? e.g. 
```
`Js_of_ocaml <http://ocsigen.org/js_of_ocaml/>`_
```